README.md

```diff
@@ -1,6 +1,10 @@
-# Service Worker Toolbox
+# Deprecated
```

Comment:
Suggestion: I think I'd keep the name at the top, but if you want it to stand out more you could always add HTML tags (which I believe work in GitHub markdown files) or use emoji to call attention to it (e.g. ⚠️).